### PR TITLE
stable/external-dns missing back ticks readme

### PR DESCRIPTION
**Pull request by hazurd**
signed-off-by: Karl XD <hazurd1@gmail.com>

<!--
#### Which issue this PR fixes
  - fixes missing back ticks in external-dns readme
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/external-dns/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.19.1
+version: 2.20.1
```
Comment (rafariossaa) [inline stable/external-dns/Chart.yaml]: Hi,
 As this is a small change, please update only the patch number, not the minor number.

**Review by rafariossaa — CHANGES_REQUESTED** [review]
Thanks for noticing this and sending a PR.

File: stable/external-dns/Chart.yaml (hunk 1 above)
Comment (hazurd) [reply stable/external-dns/Chart.yaml]: Committed the change to bump patch over minor.

**Comment by rafariossaa** [comment]
Hi,
  I am sorry, charts are being restructured.
  Do you mind to make this PR, to this other repository https://github.com/bitnami/charts.

  Thanks forehand.

**Comment by hazurd** [comment]
I'm really not sure what to do about the chart linting fail, it goes against what was requested as a version bump change.

**Pull request closed without merging**